pallets/gear/src/benchmarking/mod.rs

```diff
@@ -933,7 +934,7 @@ benchmarks! {
         core_processor::process::<
             Externalities,
             ExecutionEnvironment,
-        >(&block_config, context, memory_pages);
+        >(&block_config, context, (Vec::new(), 0), memory_pages);
```

Comment:
```suggestion
        >(&block_config, context, memory_pages);
```
Should be in block config